crypto/ec/ecdh_ossl.c

```diff
@@ -66,6 +66,10 @@ int ecdh_simple_compute_key(unsigned char **pout, size_t *poutlen,
     BN_CTX_start(ctx);
     x = BN_CTX_get(ctx);
     y = BN_CTX_get(ctx);
+    if (y == NULL) {
```

Comment:
It is of course not too good if at other places the error handling works different.
This is just how I fixed similar bugs in the past, and I personally have not looked
at all places, which idiom is more prevailing.